JUnit/Seasons/src/main/java/SeasonCheck.java

```diff
@@ -0,0 +1,31 @@
+import java.time.Month;
+
+public class SeasonCheck {
+    public static void main(String[] args) {
+        System.out.println(SeasonCheck.getSeason(4));
+    }
+
+    public static String getMonth(int monthNum) {
+        return String.valueOf(Month.of(monthNum));
+    }
+
+    public static String getSeason(int monthNum) {
+        if ((getMonth(monthNum) == "APRIL") || (getMonth(monthNum) == "MARCH") || (getMonth(monthNum) == "MAY")) {
```

Comment:
W takiej sytuacji o wiele lepsze będzie użycie konstrukcji switch case.

![image](https://user-images.githubusercontent.com/17989854/127784441-a1947a9e-faf2-432d-8da0-b7754b793130.png)

oraz to jest zbędne